.github/ISSUE_TEMPLATE/help.md

```diff
@@ -5,42 +5,71 @@ labels: 'type: question'
 
 ---
 
-<!-- EssentialsX help guide
+<!-- EssentialsX help request guide
 
-Don't write inside the arrows as they will be hidden when you post your issue.
+NOTE: Failure to fill out this template properly may result in your issue being
+      delayed or ignored.
 
-Want faster support? Come join our Discord server to get help sooner:
-https://discord.gg/F7gexAQ
+Don't put anything inside this block, as it will be hidden when you post your
+issue. Make sure to also not type between any arrows in the template, as this
+text will also be hidden.
 
-If you're happy to wait (or you were sent here from Discord), read on:
+Want faster support? Come join our Discord server to get help from both
+developers and community members: https://discord.gg/F7gexAQ
```

Comment:
Oftentimes Discord support is not fast since it is entirely driven by volunteers most hours of the day, so it would be good not to get people's expectations up.
```suggestion
Want an answer sooner? Join our Discord server to get help from community
members and sometimes even the developers themselves! Simply ask and wait
to see if anyone is able to help: https://discord.gg/F7gexAQ
```

Reply:
The text in this change feels clunky, and it carries pretty much the same meaning as before?

Reply:
Tried to keep most of the meaning the same, "sooner" does not imply "fast", and there is an emphasis put on community support over developer support (so people aren't expecting developers to always be available to respond). Lastly, "ask and wait" again to indicate that it may not be instant.

Could use some better wording, this was merely a suggestion.